lib/features/coin_flow/cubit/search_coin_cubit.dart

```diff
@@ -1,16 +1,47 @@
+import 'dart:developer';
+
 import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
 
 part 'search_coin_state.dart';
 
 class SearchCoinCubit extends Cubit<SearchCoinState> {
-  SearchCoinCubit() : super(const SearchCoinInitialState());
+  SearchCoinCubit()
+      : super(SearchCoinState(
+            status: CoinAnimationStatus.initial, stopwatch: Stopwatch()));
 
   static const searchDuration = Duration(milliseconds: 2000);
 
-  Future<void> searchForCoin() async {
-    emit(const SearchCoinAnimationState());
-    await Future.delayed(searchDuration);
-    emit(const SearchCoinFoundedState());
+  void startAnimation() async {
+    log('startAnimation');
```

Comment:
I think these logs can be removed, since there were added just in debugging purposes